### PR TITLE
Updates WPGRAPHQL_WOOCOMMERCE_VERSION constant to match plugin version.

### PR DESCRIPTION
**Pull request by jmotes**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [X] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [X] Make sure you are requesting to pull request from a **topic/feature/bugfix/devops branch** (right side). Don't pull request from your master!
- [X] Have you ensured/updated that CLI tests to extend coverage to any new logic. Learn how to modify the tests [here](https://woographql.com/contributing/2-local-testing).

What does this implement/fix? Explain your changes.
---------------------------------------------------
Updates WPGRAPHQL_WOOCOMMERCE_VERSION constant to match plugin version.

Does this close any currently open issues?
------------------------------------------
#677 

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------

- **WooGraphQL Version: 0.12.0
- **WPGraphQL Version: 1.13.6
- **WordPress Version: 6.1.1
- **WooCommerce Version: 7.1.1

### CODE REVIEW

**Comment by jmotes** [comment]
@kidunot89 I realized after I created this that you'd need to create yet another version in order to fix the constant. I can bump all the numbers to 0.12.1 if that would help ya.

**Review by kidunot89 — APPROVED** [review]

File: wp-graphql-woocommerce.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
 function constants() {
 	// Plugin version.
 	if ( ! defined( 'WPGRAPHQL_WOOCOMMERCE_VERSION' ) ) {
-		define( 'WPGRAPHQL_WOOCOMMERCE_VERSION', '0.11.2' );
+		define( 'WPGRAPHQL_WOOCOMMERCE_VERSION', '0.12.0' );
```
[the remark this answers is not in the document]
Comment (jmotes) [reply wp-graphql-woocommerce.php:34]: Resolved to use existing convention.

[the remark this answers is not in the document]
Comment (jmotes) [reply wp-graphql-woocommerce.php:34]: Resolved to use existing convention.

[the remark this answers is not in the document]
Comment (jmotes) [reply wp-graphql-woocommerce.php:34]: Code Climate, this is WordPress. We use tabs :P

**Review by kidunot89 — APPROVED** [review]

**Comment by kidunot89** [comment]
🤷🏿‍♂️ At least now it is correct in the codebase.

**Pull request merged**